b2luigi/batch/processes/gbasf2_utils/pickle_utils.py

```diff
@@ -28,4 +29,5 @@ def write_path_and_aliases_to_file(basf2_path, file_path):
     with open(file_path, 'bw') as pickle_file:
         serialized = serialize_path(basf2_path)
         serialized["aliases"] = get_alias_dict_from_variable_manager()
+        serialized["globaltags"] = b2conditions.globaltags
```

Comment:
The function name `write_path_and_aliases_to_file` and its description don't reflect that globaltags are also saved. To the user of the function it should be obvious what's happening. So I would either:
- add the globaltags outside this function, possibly via a separate function
- What I prefer would be to change the function name to something more general like `write_path_and_state_to_file()` and document in the docstring that aliases and basf2 paths are also stored. Here we don't have to worry about breaking backwards compatibility, since this we are still in the `0.` release numbers. 

One could alternatively add also function parameters (`save_aliases=True, save_global_tag=True`) to steer which parts of the global states are stored, but I don't think we really need this

- [ ] reflect global tag addition in function name and docs